src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/Intellisense.cs

```diff
@@ -22,13 +22,13 @@ namespace Microsoft.PowerFx.Core.Texl.Intellisense
     internal partial class Intellisense : IIntellisense
     {
         protected readonly IReadOnlyList<ISuggestionHandler> _suggestionHandlers;
-        protected readonly EnumStore _enumStore;
+        protected readonly PowerFxConfig _config;
 
-        public Intellisense(EnumStore enumStore, IReadOnlyList<ISuggestionHandler> suggestionHandlers)
+        public Intellisense(PowerFxConfig config, IReadOnlyList<ISuggestionHandler> suggestionHandlers)
```

Comment:
>Intellisense(PowerFxConfig config, [](http://example.com/codeflow?start=15&length=34)

Will this be an inssue for PAClient? 
CanvasIntellisense., DataverseIntellisense both impl it.

Reply:
Yes, updated